pkg/config/cos.go

```diff
@@ -732,6 +731,23 @@ func genBootstrapResources(config *HarvesterConfig) (map[string]string, error) {
 		bootstrapConfs[templateName] = rendered
 	}
 
+	// render addons
+	for _, templateName := range []string{
+		"harvester-seeder.yaml",
+		"nvidia-driver-toolkit.yaml",
+		"pcidevices-controller.yaml",
+		"rancher-logging.yaml",
+		"rancher-monitoring.yaml",
+		"vm-import-controller.yaml",
+	} {
```

Comment:
Just a thought as I'm looking through this - is it possible to somehow grab all the template names from the addons repo, so if a new addon is created in future, we don't need to explicitly specify its name here?

Reply:
i had made some changes to the addons side, so we are back to using just one addons file.